docs/run-nym-nodes/nodes/mixnodes.md

```diff
@@ -28,6 +34,21 @@ Remember that you can **only** use Cosmos `NYM` tokens to bond your mixnode. You
 #### Sandbox testnet
 Make sure to download a wallet and create an account as outlined above. Then head to our [token faucet](https://faucet.nymtech.net/) and get some tokens to use to bond it. 
 
+### VPS Hardware Specs
+You will need to requisition a VPS to run your mix node on. One key reason for this is that your node **must be able to send TCP data using both IPv4 and IPv6** (as other nodes you talk to may use either protocol. 
```

Comment:
I think own is a better word than requisition.

Reply:
Good point